spaceship/lib/spaceship/two_step_or_factor_client.rb

```diff
@@ -222,6 +225,7 @@ def sms_automatically_sent(response)
 
     # extracted into its own method for testing
     def ask_for_2fa_code(text)
+      raise "This code should only run in interactive mode. You can set the environment variable `SPACESHIP_2FA_SMS_CODE`." unless UI.interactive?
```

Comment:
What does this mean exactly?

Reply:
This was to complete the first job (sending the SMS code) without need of cancelling the job. It's redundant and not backward compatible, so should be removed/replaced by something else.